### ISSUE TITLE
explicit call to Suite.free in Python fails

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#315]
```
import cocoex as ex

s = ex.Suite("bbob", "", "")

s.free()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-c18e1937460e> in <module>()
----> 1 s.free()

interface.pyx in interface.Suite.free (cython/interface.c:3630)()

AttributeError: 'interface.Suite' object has no attribute '__dealloc__'
```

The method `__dealloc__` is defined in the code right after the `free` method.

**Comment by brockho** [comment]
FTR, this issue is still occurring (tested by @asmaatamna today).

**Comment by nikohansen** [comment]
Fixed with commit 19dca18aef97459e4493c7cb6810cb9cc4c473f3. The underlying problem is most likely related (and possibly intrinsic) to Cython and the implemented fix looks like a workaround.
